server/test/services/ti/TrustedIntermediaryServiceTest.java

```diff
@@ -362,91 +365,266 @@ private void setupTIAccount(
     applicant.save();
   }
 
+  private AccountModel setupTiClientAccount(String email, TrustedIntermediaryGroupModel tiGroup) {
```

Comment:
question (non-blocking): I'm used to having helper methods at the end of the file instead of in between `@Test` methods, do we enforce that on CiviForm or do we prefer to have the helper methods next to where they're used?